src/smc-webapp/frame-editors/rmd-editor/actions.ts

```diff
@@ -2,30 +2,98 @@
 R Markdown Editor Actions
 */
 
+import { Set } from "immutable";
+import { debounce } from "lodash";
 import { Actions } from "../markdown-editor/actions";
-import { convert } from "./rmd2md";
+import { convert } from "./rmd-converter";
+import { markdown_to_html_frontmatter } from "../../markdown";
 import { FrameTree } from "../frame-tree/types";
+import { redux } from "../../app-framework";
+import { change_filename_extension } from "../generic/misc";
 
 export class RmdActions extends Actions {
+  private _last_save_time: number = 0;
+
   _init2(): void {
+    super._init2(); // that's the one in markdown-editor/actions.ts
```

Comment:
This is highly confusing for me and maybe it's a bug: the line below suggests, that there is something running after "markdown". The `Actions` class is derived from the one for Markdown, and it also has a `_init2` method. So, the parent `_init2` initializes spellchecking and hooks up on change events for synchronizing. So, I think this is an oversight, and therefore I am calling the parent `_init2`. That's why spellchecking is active.